### PR TITLE
add track-record value translation tag

### LINKED ISSUE #5292: Add template tag for frontend to tap into `track_record_choices` for the PNI v4 product
We need to make sure that the `Great`, `Average`, `Needs work` and `Bad` strings end up properly localized on our product pages, which will required using the same approach as https://github.com/mozilla/foundation.mozilla.org/blob/master/network-api/networkapi/buyersguide/templatetags/yes_no.py#L19 for new `track_record_choices` field.

### PR DESCRIPTION
**Pull request by Pomax**
Closes #5292
Related PRs/issues #5295

This adds the necessary tag to ensure that company track record values end up properly localized.

/cc @KalobTaulien this should be a minimal change to https://github.com/mozilla/foundation.mozilla.org/pull/5295/files#diff-47a11420d29f9efdcce95f7e166919e3R138, just needing ` | track_record`

### CODE REVIEW

File: network-api/networkapi/buyersguide/templatetags/bg_selector_tags.py [hunk 1]
```diff
@@ -27,3 +27,22 @@ def extended_yes_no(value):
...
+@register.filter
+def track_record(value):
+    """
+    effects localization for company track records. While it might
+    seem easier to just return gettext(value), we want to be explicit
+    about the possible options, and the context in which to apply
+    this tag, rather than a generic "localize" tag.
+    """
+    if value == 'Great':
+        return gettext('Great')
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templatetags/bg_selector_tags.py:41]: @TheoChevalier does `gettext` allow passing an extra value to indicate what the translation context is?

Comment (TheoChevalier) [reply network-api/networkapi/buyersguide/templatetags/bg_selector_tags.py:41]: Yes! You need to use `pgettext()` for this, here’s an example: https://github.com/mozilla/foundation.mozilla.org/blob/master/network-api/networkapi/buyersguide/views.py#L143-L145

**Review by KalobTaulien — APPROVED** [review]
Looks good to me. Just the one comment from @Pomax and @TheoChevalier in the `bg_selector_tags.py` file that needs to be resolved.

**Comment by Pomax** [comment]
@TheoChevalier updated the code with `pgettext`,, does that phrasing work?

**Pull request merged**